lib/screenplay/stops/stop.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Screenplay.Stops.Stop do
+  @moduledoc """
+  Functions used to fetch stop data from the V3 API.
+  """
+
+  @behaviour Screenplay.Stops.Behaviour
+
+  alias Screenplay.V3Api
+
+  @impl true
+  def fetch_parent_stops(stop_ids) do
```

Comment:
In this function name we call them "parent stops", but in the other one it's "parent stations". I'd vote to make these consistent. ("parent stations" might be more aligned with how they're called elsewhere)

Reply:
Actually, this is my bad, I thought this was specifically fetching parent stations, but now I see it's just fetching stops by ID. I think a better name for this would be `fetch_by_ids` or something.

Reply:
To clarify the terminology as I understand it:

* A "stop" is one of the following depending on context:
  * a "stopping location" a.k.a. a stop with `location_type=0`
  * any entry in `stops.txt`, which includes "stopping locations", parent stations, entrances/exits, and pathway nodes
* A "parent station" is a stop with `location_type=1`. Stops that are _not_ parent stations may "have" or "belong to" a parent station. For entrances/exits and pathway nodes this is required, for "stopping locations" it is optional.
* A "child stop" is a stop that has a parent station, or depending on context, possibly just a "stopping location" that has a parent station.
  * Sometimes we use "platform" or "berth" to specifically refer to this latter case, but this is not a formal GTFS term.